src/containers/Interfaces/AlterForm.js

```diff
@@ -124,7 +126,7 @@ class AlterForm extends Component {
               key={index}
               node={node}
               index={index}
-              updateNode={updateNode}
```

Comment:
Just a small thing, it's better practice not to define handlers in the render method at all, otherwise the output is considered changed and the dom is always refreshed:

```jsx
class SlideForm extends PureComponent {
  handleSubmit = (formData) => {
    const { node, onSubmit } = this.props;
    onUpdate(node, formData);
  };

  render() {
    return (
      <Form
        onSubmit={this.handleSubmit}
      />
    );
  }
}

class AlterForm extends Component {
  handleUpdate = (node, formData) => (
    this.props.updateNode(node[entityPrimaryKeyProperty], {}, formData));
  };

  render() {
    return (
      <SlideFormNode
        node={node}
        onUpdate={this.handleUpdate}
      />
    );
  }
```

Reply:
Good call.